global.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
❔ Do the parsers for this file support comments? Wish it didn't have to be in the root but maybe we can at least use a comment to link to a description of the file format for users unfamiliar with it?